packages/dapp-high-roller/src/components/app-root/app-root.tsx

```diff
@@ -10,17 +12,32 @@ export class AppRoot {
   // TODO Tracking this issue: https://github.com/ionic-team/stencil-router/issues/77
   @Prop() match: MatchResults;
 
+  nodeProvider: any;
+  cfjs: any;
```

Comment:
We can't use typescript types?

Reply:
We can, but Rollup's imports/exports are broken and I can't use the dependencies properly. Everytime I import something into the Stencil project, I'm flooded by `[name] is not exported by` errors.